### PR TITLE
Add testing documentation, remove outdated Vagrantfile 

### PR DESCRIPTION
**Pull request by cwjohnston**

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+* Be advised that even once you have a Windows basebox built from one of the boxcutter, you may not be able to install the required version of Microsoft .Net Framework without manual intervention (e.g. attaching installation media as a shared folder and exporting the path as the value of the `SENSU_WINDOWS_DISM_SOURCE` environment variable. See [this issue on the sensu/sensu-build project](https://github.com/sensu/sensu-build/issues/149) for more details.
+
+## Testing and evaluating Sensu
+
+```
+cd examples
+gem install bundler
+bundle install
+librarian-chef install
+vagrant up
```
Comment (tas50) [inline TESTING.md]: Shouldn't this get changed if the Cheffile and the Vagrantfile are gone?

Comment (luckymike) [inline TESTING.md]: Indeed, good catch.

**Comment by cwjohnston** [comment]
@tas50 thanks for catching that :+1:

**Comment by luckymike** [comment]
LGTM

**Pull request merged**